app.js

```diff
@@ -21,19 +21,21 @@ config.loadConfig()
 		exit(1)
 	})
 
-// NOTE might have to blacklist things like robots.txt and this from /room names
+// -- NOTE -- might have to blacklist things like robots.txt and this from /room names
```

Comment:
```suggestion
// NOTE might have to blacklist things like robots.txt and this from /room names
```
Using `// NOTE` is a standard convention that most IDEs will pick up and notice, by making it a little different, some IDEs won't notice it

Reply:
Alright, I'll change it. I usually tend to use `// -- <TYPE> -- MESSAGE` To keep it cleaner.